package/package.json

```diff
@@ -44,7 +44,7 @@
 	},
 	"type": "module",
 	"peerDependencies": {
-		"astro": "^4.12.0 || ^5.0.0-beta"
```

Comment:
@Fryuni I'm surprised this range doesn't work, any idea why?

Reply:
`^5.0.0-beta` is equivalent to `>=5.0.0-beta <5.0.0`

Going out of beta is considered a breaking change